main/haproxy/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Contributor: Jeff Bilyk <jbilyk@gmail.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=haproxy
-pkgver=1.9.6
+pkgver=1.9.7
 _pkgmajorver=${pkgver%.*}
 pkgrel=0
 pkgdesc="A TCP/HTTP reverse proxy for high availability environments"
```

Comment:
license is 'GPL-2.0-or-later  LGPL-2.1-or-later'

Reply:
empty variables can be removed for brevity

Reply:
License changed, according to http://www.haproxy.org/download/1.5/doc/LICENSE